modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextModeAndSettings.vue

```diff
@@ -150,8 +149,14 @@ export default {
           return 'Submit';
         }
       }
+    },
+    hasBeenPublishedThisPageload() {
+      return (this.context.lastPublishedAt > this.mountedAt) || ((this.context.submitted && this.context.submitted.at) > this.mountedAt);
     }
   },
+  mounted() {
+    this.mountedAt = (new Date()).toISOString();
```

Comment:
Maintaining the existing idea that this text changes when a submission happens while the page is in the browser.